### PR TITLE
Cooking and Smallmob tweaks

### PR DESCRIPTION
**Pull request by NanakoAC**
Just an assorted package of little things:

Scales down the deep fryer, reducing its power usage, oil reservoir and  cooking containers. Should help with complaints about power usage

Fixes up numbers so that cookers heat up in 15 minutes as intended

Adjusts the normal size of all conbination cooking items, so its harder to reach crazy sizes

Removes some debug messages

Fixes user not being populated in get_scooped procs, which was preventing the special "X climbs onto Y" message from appearing when a mouse drags themselves onto a human

adds passtable code to girders and empty machine frames, allowing small mobs to crawl over them and projectiles to sometimes (50% chance) pass over

### CODE REVIEW

File: code/game/machinery/kitchen/cooking_machines/_appliance.dm [hunk 1]
```diff
@@ -142,6 +142,15 @@
 	set name = "Choose output"
 	set category = "Object"
 
+	if (!isliving(usr))
```
Comment (Lohikar) [inline code/game/machinery/kitchen/cooking_machines/_appliance.dm]: Probably should check that the user is conscious as well.

**Comment by NanakoAC** [comment]
Addresses most of the things in #1815 which are going to be addressed.

Cooking times are working as intended
The rolling pin sound was not caused by my cooking PR. It simply doesn't have a specially suitable sound defined. I can't find one

File: code/game/machinery/kitchen/cooking_machines/_appliance.dm (hunk 1 above)
Comment (skull132) [inline code/game/machinery/kitchen/cooking_machines/_appliance.dm]: What he said.

There's a standard set of checks for verbs like these.

File: code/game/machinery/kitchen/cooking_machines/_appliance.dm [hunk 2]
```diff
@@ -142,6 +142,15 @@
...
+		usr << "Ghosts aren't allowed to mess with cooking machines!"
+		return
+
+	if (!Adjacent(usr))
+		if (!istype(usr, /mob/living/silicon))
```
Comment (skull132) [inline code/game/machinery/kitchen/cooking_machines/_appliance.dm]: Isn't issilicon a macro/proc? Use that.

File: html/changelogs/Nanako-tweaks.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   maptweak
+#   spellcheck (typo fixes)
+#   experiment
+#################################
+
+# Your name.  
+author: N3X15
```
Comment (skull132) [inline html/changelogs/Nanako-tweaks.yml]: Changelog author name.

**Pull request merged**